new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/LoggingExporter.java

```diff
@@ -468,4 +562,69 @@ private static String execLoggingExpStatusCheck(String namespace, String labelSe
 
     return statusLine.stdout();
   }
+
+  private static ExecResult downloadWlsLoggingExporterJarsAndVerify(String jarReposUrl,
+                                                                    String jarFileName,
+                                                                    String wlsLoggingExporterArchiveLoc)
+        throws IOException, InterruptedException {
+
+    ExecResult result = null;
+    File wlsLoggingExpFile =
+        new File(wlsLoggingExporterArchiveLoc + "/" + jarFileName);
+    int i = 0;
+
+    StringBuffer getJars = new StringBuffer();
+    getJars
+        .append(" wget -P ")
+        .append(wlsLoggingExporterArchiveLoc)
+        .append(" --server-response --waitretry=5 --retry-connrefused ")
+        .append(jarReposUrl)
+        .append("/")
+        .append(jarFileName);
+    logger.info("Executing cmd " + getJars.toString());
+
+    // Make sure downloading completed
+    while (i < maxIterationsPod) {
```

Comment:
variable name maxIterationsPod is not appropriate here, also use awaitility api like we do in other cases ...

Reply:
done